newIDE/app/src/InstancesEditor/index.js

```diff
@@ -497,8 +497,8 @@ export default class InstancesEditor extends Component<Props> {
   };
 
   _onMouseMove = (x: number, y: number) => {
-    this.lastCursorX = x;
-    this.lastCursorY = y;
+    this.lastCursorX = x || 0;
+    this.lastCursorY = y || 0;
```

Comment:
I'm going to nitpick on this :P 

x and y should never be null/undefined, because the types are set to `number` (and not ?number).
So if null/undefined appeared here, the problem should be fixed beforehand, probably in `this.backgroundArea.on('mousemove', event =>`? If the event is not containing the x/y, we should fix it at this moment (so that the rest of the code can "confidently" use these).